src/components/Dashboard/TodoList.vue

```diff
@@ -310,15 +310,30 @@
                   </v-btn>
                 </template>
 
-                <!-- non-staff see no buttons for correction or conversion or restoration or Continuation Out -->
+                <!-- non-staff see no buttons for these filings (these are staff-only filings) -->
                 <template
-                  v-else-if="!isRoleStaff && (isTypeCorrection(item) || isTypeConversion(item) ||
-                    EnumUtilities.isTypeRestoration(item) || EnumUtilities.isTypeContinuationOut(item))"
+                  v-else-if="!isRoleStaff && (
+                    EnumUtilities.isTypeCorrection(item) ||
+                    EnumUtilities.isTypeConversion(item) ||
+                    EnumUtilities.isTypeRestoration(item) ||
+                    EnumUtilities.isTypeContinuationOut(item)
+                  )"
```

Comment:
Super nit picky and super small, but I guess we can alphabetize these and put them in their own getter? As in `staffOnlyFilings` getter since we might add even more conditions here in the future. Thoughts Sev?

Reply:
Good idea. Gimme a minute.

Reply:
Fixed, thank you! Resolving.